docs/dev/SHPWrite.md

```diff
@@ -20,6 +20,13 @@ SHPWrite(VARCHAR path, VARCHAR tableName, VARCHAR fileEncoding);
 
 Writes the contents of table `tableName` to a [shapefile][wiki]
 located at `path`.
+
+`tablename` can be either:
+
+* the name of an existing table,
+* the result of a selection (`SELECT` instruction which has to be written between simple quote `' '`).
```

Comment:
of a query

Reply:
instruction which has to be written between simple quote and parenthesis

Reply:
Ok